src/ralph/virtual/admin.py

```diff
@@ -52,11 +55,25 @@ class VirtualServerTypeForm(RalphAdmin):
 
 
 class VirtualServerForm(RalphAdminForm):
+    HYPERVISOR_TYPE_ERR_MSG = 'Hypervisor must be one of \
+                                   DataCenterAsset or CloudHost'
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if 'parent' in self.fields:
             self.fields['parent'].required = True
 
+    def clean_parent(self):
+        value = self.cleaned_data.get('parent')
+        self._validate_parent_type(value)
+        return value
+
+    def _validate_parent_type(self, value):
+        dc_asset = ContentType.objects.get_for_model(DataCenterAsset)
```

Comment:
you might use `ContentType.get_for_models`

Reply:
Done.